src/maintainer/conda_forge_yml.rst

```diff
@@ -248,6 +249,16 @@ Currently only:
     linux_ppc64le:
       enabled: False
 
+os_version
+----------
+This key is used to set the OS versions for ``linux_*`` platforms. Valid entries map a linux platform and arch to either ``cos6``
+or ``cos7``. Currently ``cos6`` is the default. Here is an example that enables CentOS 7 on ``linux-64`` builds
```

Comment:
``cos6`` is the default for ``linux-64``

Reply:
Good catch.